neqo-crypto/Cargo.toml

```
@@ -17,6 +17,7 @@ workspace = true
 
 [dependencies]
 enum-map = { workspace = true }
+flate2 = "1.1.1"
```

Firefox is at a different version. Also pin in `Cargo.lock`.

```suggestion
flate2 = "1.0.30"
```